src/plots/wheelPlot.ts

```diff
@@ -0,0 +1,228 @@
+      if (this.innerRadius() != null && this.innerRadius().scale != null) {
+        this.innerRadius().scale.range([0, radiusLimit]);
+      }
+      if (this.outerRadius() != null && this.outerRadius().scale != null) {
+        this.outerRadius().scale.range([0, radiusLimit]);
+      }
+      return this;
+    }
+
+    protected _createDrawer(dataset: Dataset) {
+      return new Plottable.Drawers.Arc(dataset);
+    }
+
+    public entities(datasets = this.datasets()): PlotEntity[] {
+      let entities = super.entities(datasets);
+      entities.forEach((entity) => {
+        entity.position.x += this.width() / 2;
+        entity.position.y += this.height() / 2;
+      });
+      return entities;
+    }
+
+    protected _getDataToDraw() {
+      let dataToDraw = super._getDataToDraw();
+      if (this.datasets().length === 0) { return dataToDraw; }
+      let startAngleAccessor = Plot._scaledAccessor(this.startAngle());
+      let endAngleAccessor = Plot._scaledAccessor(this.endAngle());
+      let innerRadiusAccessor = Plot._scaledAccessor(this.innerRadius());
+      let outerRadiusAccessor = Plot._scaledAccessor(this.outerRadius());
+      let ds = this.datasets()[0];
+      let data = dataToDraw.get(ds);
+      let filteredData = data.filter((d, i) =>
+        Plottable.Utils.Math.isValidNumber(startAngleAccessor(d, i, ds)) &&
+        Plottable.Utils.Math.isValidNumber(endAngleAccessor(d, i, ds)) &&
+        Plottable.Utils.Math.isValidNumber(innerRadiusAccessor(d, i, ds)) &&
+        Plottable.Utils.Math.isValidNumber(outerRadiusAccessor(d, i, ds)));
+      dataToDraw.set(ds, filteredData);
+      return dataToDraw;
+    }
+
+    protected _propertyProjectors(): AttributeToProjector {
+      let attrToProjector = super._propertyProjectors();
+      let innerRadiusAccessor = Plot._scaledAccessor(this.innerRadius());
+      let outerRadiusAccessor = Plot._scaledAccessor(this.outerRadius());
+      let startAngleAccessor = Plot._scaledAccessor(this.startAngle());
+      let endAngleAccessor = Plot._scaledAccessor(this.endAngle());
+      attrToProjector["d"] = (datum: any, index: number, ds: Dataset) => {
+        let startAngle = startAngleAccessor(datum, index, ds);
+        let endAngle = endAngleAccessor(datum, index, ds);
+        if (endAngle < startAngle) {
+          endAngle += (Math.floor((startAngle - endAngle) / (Math.PI * 2)) + 1) * Math.PI * 2;
+        }
+        return d3.svg.arc().innerRadius(innerRadiusAccessor(datum, index, ds))
+                           .outerRadius(outerRadiusAccessor(datum, index, ds))
+                           .startAngle(startAngle)
+                           .endAngle(endAngle)(datum, index);
+      };
+      return attrToProjector;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for the start angle.
+     */
+    public startAngle<T>(): AccessorScaleBinding<T, number>;
+    /**
+     * Sets the start angle to a constant number or the result of an Accessor<number>.
```

Comment:
Currently this method expects radians, but I wonder if we should expect degrees instead? `angle()` on `Label` and `tickLabelAngle()` on `Axes.Category` take degrees, for example (but not all possible values). Maybe worth an RFC? Or follow up on the existing thread.

Either way, we should specify the expected units in the docs.